project-p3-forms-master/code/index.html

```diff
@@ -0,0 +1,64 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8" />
+    <title>Form-draft</title>
+    <link rel="stylesheet" href="./style.css"/>
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  </head>
+
+  <body>    
+    <h2>Mehr Infos?</h2>
+    <form>
+      <action="https://httpbin.org/anything" method="POST"></action>
+      <label for="Name">Name</label>
+      <input type="text" id="Name" name="Name"><br><br>
+      <label for="Email">Email</label>
+      <input type="email" id="Email" name="Email" value="xy@gmx.ch"><br><br>
```

Comment:
Instead of `value`, you could change this to `placeholder` so that the user doesn't have to remove the value before writing their own